extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -582,7 +599,7 @@ static public function setVarById( $cv_variable_id, $city_id, $value, $reason=nu
 			/**
 			 * use master connection for changing variables
 			 */
-			$variable = self::loadVariableFromDB( $cv_variable_id, false, $city_id, true );
+			$variable = static::loadVariableFromDB( $cv_variable_id, false, $city_id, true );
 			$oldValue = isset( $variable->cv_value ) ? $variable->cv_value :  false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)